### ISSUE TITLE
Improve SimplyE logo labelling for screen readers

### ISSUE THREAD

**Issue by seanredmond** [NYPL-Simplified/web-reader-landing-page#147]
In the current markup, the SimplyE logo in the header is misread as "sim-ply" (rhymes with "cry"). If it is spelled with a space between "simply" and "e", (i.e. "Simply E") it will be read correctly.

I think the markup could also be simplified by replacing the `aria-labelledby` pointer to the `title`

    <svg ... aria-labelledby="SimplyE-logo" class="simply-logo">
        <title id="SimplyE-logo">SimplyE</title>
    ...  
    
with an `aria-label` attribute.  I don't think the SVG needs a title.

    <svg ... aria-label="Simply E" class="simply-logo">

**Comment by EdwinGuzman** [comment]
Now that the SimplyE logo and "Subway Library" are all one svg, do you think the aria-label should be "Simply E Subway Library"? Or something to include Subway Library?

**Comment by EdwinGuzman** [comment]
@jce1028 currently being reviewed in PR #186 . Going to update the milestone.
@seanredmond can you take a look?
